src/components/streams/TrendingStreamsContainer.js

```diff
@@ -6,7 +6,7 @@ import TrendingStreams from './TrendingStreams'
 import { loadTrendingStreams } from '../../actions/stream'
 
 function mapStateToProps (state, ownProps) {
-  const streams = at(state.stream, state.stream.trending)
+  const streams = at(state.stream, state.stream.trending || [])
```

Comment:
This particular change makes sure that empty streams are _properly_ handled :) Before it would cause an error.